plugin/src/sbt-test/scalafmt-sbt/sbt/build.sbt

```diff
@@ -5,6 +5,16 @@ ThisBuild / fork := true
 
 lazy val p123 = project
   .in(file("."))
+  .settings(
```

Comment:
what does this have to do with sbt2?

Reply:
add new `updateLastModified` inputTask for root project.

https://www.scala-sbt.org/2.x/docs/en/changes/migrating-from-sbt-1.x.html

> In sbt 1.x bare settings were project settings that applied only to the root subproject. In sbt 2.x, the bare settings in build.sbt are common settings that are injected to all subprojects.